packages/mobile/src/components/balance-pill/BalancePill.tsx

```diff
@@ -0,0 +1,50 @@
+import type { ReactNode } from 'react'
+
+import { Flex, Text } from '@audius/harmony-native'
+import Skeleton from 'app/components/skeleton'
+import { makeStyles } from 'app/styles'
+import { spacing } from 'app/styles/spacing'
+
+const useStyles = makeStyles(({ spacing, palette }) => ({
+  amount: {
+    paddingRight: spacing(1.5),
```

Comment:
any reason this pr doesn't seem to be using the token values from `spacing`?   `const { spacing } = useTheme()`?

Reply:
old code. changing now that i'm moving to harmony-native